tests/storage/test_storage_stats.py

```diff
@@ -0,0 +1,14 @@
+from lamindb_setup.core.upath import get_stat_dir_s3, UPath
+
+
+def test_get_stat_dir_s3():
+    import re
+
+    message = UPath(
+        "s3://lamindb-dev-datasets/iris_studies/study0_raw_images", anon=True
+    ).view_tree()
+    num_objects_view_tree = re.search(r"(\d+) files", message).group(1)
+    _, _, _, num_objects_stat = get_stat_dir_s3(
+        UPath("s3://lamindb-dev-datasets/iris_studies/study0_raw_images", anon=True)
+    )
+    assert num_objects_stat == int(num_objects_view_tree)
```

Comment:
Hmmmm. 🤔

Reply:
Instead of using regex, cold you save `n_files` as a private attribute in `UPath` and then test for it?

I also suggest you rename it to `n_objects`.